src/NHibernate/Linq/Visitors/SelectClauseVisitor.cs

```diff
@@ -34,7 +34,7 @@ public IEnumerable<HqlExpression> GetHqlNodes()
 			return _hqlTreeNodes;
 		}
 
-		public void VisitSelector(Expression expression)
+		public void VisitSelector(Expression expression, bool isSubQuery = false)
```

Comment:
This is binary breaking and cannot be accepted in a patch or minor version. A new overload is required instead.

Reply:
Fixed